<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QDialog" name="SettingsForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>369</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>369</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>369</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="methodS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Общие</string>
   </property>
   <widget class="QComboBox" name="netValue">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="methodLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод</string>
    </property>
   </widget>
   <widget class="QComboBox" name="methodValue">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>ОМК</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="netLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Решетка</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="weights">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Эвристические коэфициенты</string>
   </property>
   <widget class="QDoubleSpinBox" name="weightsFILFREE">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.400000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weightsFOBFOB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.600000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="hydroFobLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="hydroFobLabel_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weightsFILFIL">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.400000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weightsFOBFREE">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weightsFILFOB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="weightsFOBFIL">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.150000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ants">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Муравьи</string>
   </property>
   <widget class="QSpinBox" name="antsCount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="threadsCount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="countThreadLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество потоков</string>
    </property>
   </widget>
   <widget class="QLabel" name="countAntLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество муравьев</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="traces">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Следы</string>
   </property>
   <widget class="QDoubleSpinBox" name="traceCoef">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="traceCoefLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Коеффициент стадности</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="traceEvoparationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Испарение</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="traceEvoparation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.600000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="elitismLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ограничение элитарности</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="elitism">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MMAS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXP</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="traceMaxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимум</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="traceMax">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="traceMin">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="traceMinLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимум</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
